debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -39,6 +40,13 @@
     protected static final String DATABASE_INCLUDE_LIST_ALREADY_SPECIFIED_ERROR_MSG = "\"database.include.list\" is already specified";
     protected static final String DATABASE_WHITELIST_ALREADY_SPECIFIED_ERROR_MSG = "\"database.whitelist\" is already specified";
 
+    protected static final Pattern PATTERN_SPILT = Pattern.compile(",");
+
+    protected static final Pattern FIELD_EXCLUDE_LIST_PATTERN = Pattern.compile("^(?![.])([*\\w]+.[\\w]+.[\\w]+)+(.[\\w]+)*$");
+    protected static final String QUALIFIED_FIELD_EXCLUDE_LIST_PATTERN = "<databaseName>.<collectionName>.<fieldName>.<nestedFieldName>";
```

Comment:
Can these regexes get simpler? As I checked it currently accepts `inventory` or `inventory.name` values which it shouldn't. Any suggestions for modification?